eng/native/configureplatform.cmake

```diff
@@ -315,10 +320,10 @@ if (CLR_CMAKE_TARGET_OS STREQUAL CLR_CMAKE_HOST_OS)
         endif()
     endif()
 else()
-    if(NOT (CLR_CMAKE_HOST_OS STREQUAL Windows_NT))
+    if(NOT (CLR_CMAKE_HOST_OS STREQUAL Windows_NT) AND NOT (CLR_CMAKE_HOST_OS STREQUAL Linux AND CLR_CMAKE_TARGET_OS STREQUAL FreeBSD))
```

Comment:
This change should be removed.  It is a result of the incorrect setting of  `CLR_CMAKE_HOST_OS` above.